assets/main.js.liquid

```diff
@@ -81,33 +81,189 @@ Site.Header = {
 };
 
 Site.Product = {
+  minWidth: 720, // minWidth for fixed header layout
+  numberRelatedToPick: 4, // number of related products to show
   init: function() {
     var _this = this;
 
-    if ($('#product-gallery').length) {
-      _this.setGalleryWidth();
+    _this.$container = $('.container'); // used by toggleFixed() on scroll
+
+    _this.$productHeader = $('#product-header');
+    _this.$productGallery = $('#product-gallery');
+
+    // imagesLoaded hasn't finished yet
+    _this.loadedImages = false;
+
+    if (_this.$productGallery.length) {
+      // We have product images, so we do the fancy layout
+      _this.handleLayout();
     }
 
     if ($('#related-products').length) {
+      // Pick a few Related Products to display
```

Comment:
when would this not return true?

Reply:
see `snippets/related-products`. there are some conditions for that elem to appear